src/txn/txn.c

```diff
@@ -1779,6 +1779,14 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
                     break;
                 }
 
+                /*
+                 * For now just confirm that each operation has a weak hazard pointer and clear it
+                 * before proceeding.
+                 */
+                if ((op->type == WT_TXN_OP_BASIC_ROW || op->type == WT_TXN_OP_INMEM_ROW) &&
```

Comment:
I guess for now we're not doing this for column store?

Reply:
Based on the cherry-pick we are working from, that is correct. Let's discuss this when we meet.

Reply:
That will be my preference - Let's worry about col store later, the POC also doesn't cover it. We should have a discussion about whether or not we will be supporting column stores and to what capacity. Also, need to figure out where have we reached wrt column stores supporting timestamps.